.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -1,6 +1,6 @@
 name: Documentation
 description: Ask / Report an issue related to the documentation.
-title: "DOC: <description>"
+title: "DOC: Report an issue"
```

```suggestion
title: "DOC: <description>"
```

It is intentional to leave "`<description>`", as it suggests users to replace this with an actual description.